drivers/at86rf215/Kconfig

```diff
@@ -0,0 +1,34 @@
+# Copyright (c) 2020 Freie Universitaet Berlin
+#
+# This file is subject to the terms and conditions of the GNU Lesser
+# General Public License v2.1. See the file LICENSE in the top level
+# directory for more details.
+#
+menuconfig KCONFIG_MODULE_AT86RF215
+    bool "Configure AT86RF215 driver"
+    depends on MODULE_AT86RF215
+    help
+        Configure the AT86RF215 driver using Kconfig.
+
+if KCONFIG_MODULE_AT86RF215
+
+config AT86RF215_USE_CLOCK_OUTPUT
+    bool "Enable/Disable Onchip Clock Source"
+    default n
+    help
+        Enable this to set the clock output of the AT86RF215
+        as the clock source on the board.
```

Comment:
```suggestion
        Enable this to enable the clock output of the AT86RF215 chip.
        This way it can be used as a clock source in place of a separate crystal.
        You also have to enable this if you want to measure the clock frequency
        for trimming.
```